lib/freight/conf.sh

```diff
@@ -6,18 +8,23 @@ VARLIB="/var/lib/freight"
 VARCACHE="/var/cache/freight"
 
 # Default architectures.
+# shellcheck disable=SC2034
```

Comment:
Do you want to ignore this check in the whole file?  I think moving the disable comment to immediately after the shebang will do so.